SSA/Projects/InstCombine/Base.lean

```diff
@@ -121,10 +121,14 @@ def reprWithoutFlags (op : MOp.BinaryOp) (prec : Nat) : Format :=
     | .srem   => "srem"
     | .add ⟨false, false⟩ => "add"
     | .add ⟨nsw, nuw⟩    => toString f!"add {nsw} {nuw}"
-    | .mul    => "mul"
-    | .sub    => "sub"
-    | .sdiv   => "sdiv"
-    | .udiv   => "udiv"
+    | .mul ⟨false, false⟩ => "mul"
+    | .mul ⟨nsw, nuw⟩    => toString f!"mul {nsw} {nuw}"
+    | .sub ⟨false, false⟩ => "sub"
+    | .sub ⟨nsw, nuw⟩    => toString f!"sub {nsw} {nuw}"
+    | .sdiv ⟨false⟩ => "sdiv"
+    | .sdiv ⟨true⟩ => "sdiv exact"
+    | .udiv ⟨false⟩  => "udiv"
+    | .udiv ⟨true⟩  => "udiv exact"
```

Comment:
Here, the previous alignment is broken. Either reduce the whitespace to one character or realign, I guess. Also, does the printing work

Reply:
Thanks for the notice, I made a new commit to fix the alignment